wasm/imports.go

```diff
@@ -0,0 +1,104 @@
+package wasm
+
+import (
+	"errors"
+	"fmt"
+)
+
+// ImportEntry describes an import statement in a Wasm module.
+type ImportEntry struct {
+	// module name string
```

Comment:
Yeah, I was just following the names given by the spec. `Name` would be a better prefix.